### ISSUE TITLE
Documentation should be accessible

### ISSUE THREAD

**Issue by karaboldt** [unizin/udp-docs#30]
Part of Unizin's commitment to data standards is an accessibility-first methodology that includes a11y as an important part in the development process, rather than as an afterthought.

The current documentation is largely accessible (all menus are visible and expanded, etc.), but lacks aria roles. Other opportunities may be discovered upon deeper assessment of WCAG standards.

**Comment by karaboldt** [comment]
n/a
